specification/quota/resource-manager/readme.md

```diff
@@ -26,14 +26,24 @@ These are the global settings for the Quota API.
 
 ``` yaml
 openapi-type: arm
-tag: package-2024-10-15-preview
+tag: package-preview-2024-12-18
 ```
 
 ``` yaml !$(csharp)
 modelerfour:
   flatten-models: false
 ```
 
+### Tag: package-preview-2024-12-18-preview
+
+These settings apply only when `--tag=package-preview-2024-12-18-preview` is specified on the command line.
+
+```yaml $(tag) == 'package-preview-2024-12-18-preview'
```

Comment:
your tag name has two `-preview` which is not equal to the default tag you changed in line 29.